src/Microsoft.AspNet.Mvc.Localization/project.json

```diff
@@ -0,0 +1,31 @@
+﻿{
+  "description": "Contains the localizers for ASP.NET MVC.",
```

Comment:
@danroth27 / @DamianEdwards can you help come up with a more complete package description?